app/models/samples/json/model_shoe_box.json

```diff
@@ -0,0 +1,614 @@
+                    "conductivity": 0.16,
+                    "density": 800.0,
+                    "specific_heat": 1090.0,
+                    "thermal_absorptance": 0.9,
+                    "solar_absorptance": 0.5,
+                    "visible_absorptance": 0.5
+                },
+                {
+                    "type": "EnergyMaterial",
+                    "name": "Generic Acoustic Tile",
+                    "roughness": "MediumSmooth",
+                    "thickness": 0.02,
+                    "conductivity": 0.06,
+                    "density": 368.0,
+                    "specific_heat": 590.0,
+                    "thermal_absorptance": 0.9,
+                    "solar_absorptance": 0.3,
+                    "visible_absorptance": 0.3
+                }
+            ]
+        }
+    },
+    "rooms": [
+        {
+            "type": "RoomAbridged",
+            "name": "SimpleShoeBoxZone",
+            "name_original": "Simple Shoe Box Zone",
+            "properties": {
+                "type": "RoomProperties",
+                "energy": {
+                    "type": "RoomEnergyPropertiesAbridged",
+                    "program_type": null,
+                    "construction_set": "Shoe Box Construction Set",
+                    "people": null,
+                    "lighting": null,
+                    "electric_equipment": null,
+                    "gas_equipment": null,
+                    "infiltration": null,
+                    "ventilation": null
+                }
+            },
+            "faces": [
+                "SimpleShoeBoxZone_Bottom",
+                "SimpleShoeBoxZone_Front",
+                "SimpleShoeBoxZone_Right",
+                "SimpleShoeBoxZone_Back",
+                "SimpleShoeBoxZone_Left",
+                "SimpleShoeBoxZone_Top"
+            ]
+        }
+    ],
+    "faces": [
+        {
+            "type": "FaceAbridged",
+            "name": "SimpleShoeBoxZone_Bottom",
+            "name_original": "Simple Shoe Box Zone_Bottom",
+            "geometry": {
+                "type": "Face3D",
+                "boundary": [
+                    [
+                        0.0,
+                        0.0,
+                        0.0
+                    ],
+                    [
+                        0.0,
+                        10.0,
+                        0.0
+                    ],
+                    [
+                        5.0,
+                        10.0,
+                        0.0
+                    ],
+                    [
+                        5.0,
+                        0.0,
+                        0.0
+                    ]
+                ]
+            },
+            "properties": {
+                "type": "FaceProperties",
+                "energy": {
+                    "type": "FaceEnergyPropertiesAbridged"
+                }
+            },
+            "face_type": {
+                "type": "Floor"
```

Comment:
Is face_type a dictionary? ` "face_type": "Floor",`

Reply:
It is implemented as a dictionary and not a string because, for some face types (like AirWalls), we may want to store some other properties within the face_type instance.  For example, AirWalls might have additional parameters like the amount of air mixing or whether the air wall should be removed for daylight simulation.  For now, do not worry about this case, @tanushree04 , and just implement them as dictionaries with a single `type` parameter.  We can discuss more in our next meeting.

Reply:
Okay. So each face_type would have its own schema?